harness/propertyHelper.js

```diff
@@ -1,5 +1,6 @@
 
 function isConfigurable(obj, name) {
+    var desc = Object.getOwnPropertyDescriptor(obj, name);
```

Comment:
If you click the link, you'll note that that file has a single line, `verifyConfigurable`, yet @anba says that file has to run last.